axelrod/match_generator.py

```diff
@@ -64,10 +64,10 @@ def build_match_chunks(self, noise=0):
         A generator that returns player index pairs and match parameters for a
         round robin tournament.
 
-        Parameters
+        parameters
         ----------
         noise : float, 0
-            The probability that a player's intended action should be flipped
+            the probability that a player's intended action should be flipped
 
         Yields
```

Comment:
Not sure what that means but I'm also puzzled by my change... I have no idea how I did that and will blame my editor... :)